### PR TITLE
Update portal.json

### PR DESCRIPTION
**Pull request by echobilisim3421**
The person who did the translation included their name and surname, which was a mistake. This has been corrected.

### CODE REVIEW

**Comment by cathysarisky** [comment]
Please see previous comments about this change here: https://github.com/TryGhost/Ghost/pull/21440

File: ghost/i18n/locales/tr/portal.json [hunk 1]
```diff
@@ -95,8 +95,8 @@
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "Haber bülteni göndermeye çalışırken kalıcı bir hata alınması durumunda, hesapta e-postalar devre dışı bırakılır.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "E-posta istemcinizde {{senderEmail}} adresini kişi listenize ekleyin. Bu, posta sağlayıcınıza bu adresten gönderilen e-postaların güvenilir olması gerektiğini bildirir.",
     "Invalid email address": "Geçersiz e-posta adresi",
-    "Jamie Larson": "Ahmet Yılmaz",
-    "jamie@example.com": "ahmet@example.com",
+    "Jamie Larson": "Ad Soyad",
+    "jamie@example.com": "kullanıcıadı@example.com",
```
Comment (cathysarisky) [inline ghost/i18n/locales/tr/portal.json:99]: See https://github.com/TryGhost/Ghost/pull/21440

**Review by cathysarisky — CHANGES_REQUESTED** [review]

File: ghost/i18n/locales/tr/portal.json (hunk 1 above)
Comment (cathysarisky) [reply ghost/i18n/locales/tr/portal.json:99]: See https://github.com/TryGhost/Ghost/pull/21440#discussion_r1852660737

**Comment by 9larsons** [comment]
Please stop submitting duplicates.

**Pull request closed without merging**